src/py42/sdk/__init__.py

```diff
@@ -134,25 +163,121 @@ def securitydata(self):
             * Security plan information
 
         Returns:
-            :class:`py42.modules.securitydata.SecurityModule`
+            :class:`py42.services.securitydata.SecurityDataClient`
         """
-        return self._sdk_dependencies.security_module
+        return self._clients.securitydata
 
     @property
     def detectionlists(self):
         """A collection of properties each containing methods for managing specific detection
         lists, such as departing employees.
 
         Returns:
-            :class:`py42.modules.detectionlists.DetectionListsModule`
+            :class:`py42.services.detectionlists.DetectionListsClient`
         """
-        return self._sdk_dependencies.detection_lists_module
+        return self._clients.detectionlists
 
     @property
     def alerts(self):
         """A collection of methods related to retrieving and updating alerts rules.
 
         Returns:
-            :class:`py42.modules.alertrules.AlertRulesModule`
+            :class:`py42.services.alertrules.AlertRulesClient`
         """
-        return self._sdk_dependencies.alerts_module
+        return self._clients.alerts
+
+
+def _init_services(main_connection, main_auth):
```

Comment:
all of this configuration lives in the same file as the SDK object itself now, I think that makes it much easier to visualize the hierarchy of how everything is composed.